packages/integrations/image/src/types.ts

```diff
@@ -1,5 +1,6 @@
-export type { Image } from '../components/index';
-export * from './index';
+/// <reference path="../types/astro-jsx.d.ts" />
```

Comment:
Is there no way to refer to the `astro-jsx.d.ts` inside Astro instead of copying it here?

Reply:
I couldn't find one 🤷

Not sure if it was just the monorepo striking again, but I had assumed `path="node_modules/astro/astro-jsx.d.ts"` would have done the trick but it didn't resolve

Reply:
This doesn't work? It probably won't in the monorepo, but should outside of it.

```
/// <reference path="astro/astro-jsx.d.ts" />
```

Reply:
Ah ha!  I didn't realize `<reference type="astro/astro-jsx">` was a separate syntax, that resolves it similar to `import` 🎉